odoo/tools/mail.py

```diff
@@ -290,6 +292,22 @@ def is_html_empty(html_content):
     tag_re = re.compile(r'\<\s*\/?(?:p|div|span|br|b|i|font)(?:(?=\s+\w*)[^/>]*|\s*)/?\s*\>')
     return not bool(re.sub(tag_re, '', html_content).strip())
 
+def parse_inline_template(text):
```

Comment:
Linting: Would move to rendering tools (a bit harsh to hide it between email regexes)